setup.py

```diff
@@ -49,6 +49,6 @@
     test_suite="tests",
     tests_require=test_requirements,
     url="https://github.com/notificationapi-com/notificationapi_python_server_sdk",
-    version="0.0.7",
+    version="0.0.8",
```

Comment:
I thought I have put this in readme, but I haven't. Use this installed package to bump the versions:

I think it was something like: `bump2version major/minor/patch`